attributes/events/listener.md

```diff
@@ -0,0 +1,35 @@
+# Listener
+
+The `Listener` attribute allows you to register a listener function for model events.
+
+A more convenient way of registering a listener function "replacing" laravels event closures for eloquent models, more info (Laravel Docs: Eloquent Closures)[https://laravel.com/docs/10.x/eloquent#events-using-closures]
+
+## Usage
+
+```php
+use Illuminate\Database\Eloquent\Model;
+use WendellAdriel\Lift\Attributes\Events\Listener;
+use WendellAdriel\Lift\Lift;
+
+final class Product extends Model
+{
+    use Lift;
+
+
+    #[Listener]
+    public function onCreated(Product $product) {
+    	Log::info("Product {$product->name} has been created.");
+    }
+}
+```
+
+## API
+
+```php
+#[Listener(event: 'created', queue: true)]
+```
+
+Properties:
```

Comment:
Put this and below in text format not list, because you don't need to add the name or the type of the parameters since it's already described above in the API example.